kea-vscode/src/views/pull-request-contents/commits/local-file-tree-node.ts

```diff
@@ -0,0 +1,51 @@
+import * as vscode from "vscode";
+import { ILocalGitRepository, LocalCommit } from "../../../git/local-git-repository";
+
+/**
+ * Represents a file in a local commit tree.
+ */
+export class LocalFileTreeNode {
+  #contextValue = "localFile";
+  #iconPath = new vscode.ThemeIcon("file");
+  
+  fileName: string;
+  filePath: string;
+  status: string;
+  #localGitRepo: ILocalGitRepository;
+  #commit: LocalCommit;
+  #workspaceFolder: vscode.WorkspaceFolder;
+  
+  constructor(localGitRepo: ILocalGitRepository, commit: LocalCommit, workspaceFolder: vscode.WorkspaceFolder, filePath: string, status: string) {
```

Comment:
Please show the correct icons for the files in the tree.

Reply:
The icons shown should be of the language - e.g., the TypeScript icon for .ts files.